main.go

```diff
@@ -100,6 +102,10 @@ func main() {
 		if scope != "" {
 			q.Set("scope", scope)
 		}
+		if optionalScope != "" {
+			optionalScope = strings.ReplaceAll(optionalScope, " ", "%20")
```

Comment:
```suggestion
			optionalScope = url.QueryEscape(optionalScope)
```

Reply:
let me double check something on this real quick, because we might not need either of these - we are escaping these later.

Reply:
yeah, removing this altogether.. i added this during some troubleshooting when i thought i was taking crazy pills and then solved it a different way.